force_nevergrad/engine/old/nevergrad_optimizer_engine.py

```diff
@@ -39,12 +39,10 @@ def _create_instrumentation_variable(parameter):
         """ Create nevergrad.variable from `MCOParameter`. Different
```

Comment:
Note: I'm not sure why but this file is not being picked up by the coverage assessment. Since the unit tests have been removed, I would expect this to cause codecov to complain and we don't really want untested code hanging around in our repos anyway.

I would recommend either removing it completely if we are confident that the new MCO classes are providing all functionalities, or if not, restoring it back in the `force_nevergrad/engine/` directory, along with the previously removed unit tests. I don't think `NevergradOptimizerEngine` it is being imported by any external plugin, so we won't be breaking anything either way.